HousingRepairsOnlineApi/Helpers/RepairQueryHelper.cs

```diff
@@ -31,14 +31,15 @@ public FeedIterator<Repair> GetRepairSearchIterator(string repairType, string pr
             return result;
         }
 
-        public FeedIterator<Repair> GetRepairSearchIterator(IEnumerable<string> repairTypes, string postcode, string repairId)
+        public FeedIterator<Repair> GetRepairSearchIterator(IEnumerable<string> repairTypes, string postcode, string repairId, bool includeCancelled = false)
         {
             var repairTypesUppercase = repairTypes.Select(x => x.ToUpperInvariant());
 
             var query = cosmosContainer.Container.GetItemLinqQueryable<Repair>().Where(x =>
                     x.Id.ToUpper() == repairId.ToUpper()
                     && x.Postcode.Replace(" ", "").ToUpper() == postcode.Replace(" ", "").ToUpper()
-                    && repairTypesUppercase.Contains(x.RepairType.ToUpper()))
+                    && repairTypesUppercase.Contains(x.RepairType.ToUpper())
+                    && (includeCancelled || (x.Status.ToUpper() != RepairStatus.Cancelled.ToUpper())))
```

Comment:
Consider if this filtering should also be part of the [other](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/59/files#diff-96659e09d0811df42f48f09a6c64e33a59085fd88914d0fcf123af899b83dd9cR22) query.